OrbitGl/TimeGraph.cpp

```diff
@@ -730,7 +734,10 @@ void TimeGraph::DrawTracks(GlCanvas* canvas, PickingMode picking_mode) {
     if (track->GetType() == Track::kThreadTrack) {
       auto thread_track = std::static_pointer_cast<ThreadTrack>(track);
       int32_t tid = thread_track->GetThreadId();
-      if (tid == SamplingProfiler::kAllThreadsFakeTid) {
+      if (tid == SamplingProfiler::kAllTracepointsFakeTid) {
+        thread_track->SetName("all tracepoints");
+        thread_track->SetLabel("all tracepoints");
```

Comment:
As suggested offline, I would go for "All tracepoint events".

Reply:
Of course! Thank you!